archdoc/chap-cheri-riscv.tex

```diff
@@ -477,16 +477,40 @@ \subsection{Sealed capabilities}
   \footnotesize
   \begin{tabular}{|c|c|c|c|}
     \hline
-    \asm{cs1} & \asm{cd} & Used for & Valid \cotype{}s \\
+    \asm{cs1} & \asm{cd} (out \cotype{}s) & Used for & Valid \asm{cs1} \cotype{}s \\
```

Comment:
It might be clearer to have another column for the output otype or maybe even a separate (smaller) table? We should make clear that the same behaviour applies to `cjal`.

Reply:
Separate table it is, I think.